lib/DDGC/Schema/Result/User.pm

```diff
@@ -4,6 +4,8 @@ package DDGC::Schema::Result::User;
 
 use Moo;
 extends 'DDGC::Schema::Result';
+with 'DDGC::Schema::Role::Result::User::Subscription';
+
```

Comment:
I guess another way to make these classes shorter is to put the column definition all on one line.  I find that a lot more readable when looking at the schema definition.  But it usually violates the 80 char limit rule.  Which is not cool.  My justification is that readability is more important and rules are made to be broken.  Definitely a personal preference thing.  :)